community/OM-Operator-Management/cnv-samples/cnv-operator-install/policy-mtv.yaml

```diff
@@ -0,0 +1,164 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: openshift-virtualization-mtv
+  namespace: default
+spec:
+  remediationAction: enforce
```

Comment:
We prefer policies in the collection to be in `inform` mode, then users can adjust that themselves. (This prevents users from accidentally installing a huge number of resources at once if they applied the entire collection or something)

Reply:
I guess my counter to that is this policy is to deploy and it's keyed of a very specific label to do just this. Install CNV and MTV. So there is no way they would have this label in the env. And I wanted it as simple as you apply the policy and start adding labels. I don't want them to have to understand the policy if they don't have too.


What do you think?

Reply:
That's a good point. You can keep it as `enforce`.